src/components/Post/Post.js

```diff
@@ -0,0 +1,136 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const StyledPost = styled.div`
+  background-color: palevioletred;
+`;
+
+const Accordion = styled.button`
+  background-color: #fff;
+  font-family: 'Raleway', sans-serif;
+  font-weight: 600;
+  font-size: 14px;
+  color: #444;
+  cursor: pointer;
+  padding: 18px;
+  width: 100%;
+  text-align: left;
+  border: none;
+  outline: none;
+  transition: 0.2s;
+  position:relative;
+
+  &:after {
+    font-family: 'Lato', sans-serif;
+    font-weight: 300;
+    content: ${props => (props.isActive ? '-' : '+')};
+    font-size: ${props => (props.isActive ? '42px' : '27px')};
+    color: #777;
+    float: right;
+    margin-left: 5px;
+    top: 50%;
+    transform: translateY(-50%);
+    position: absolute;
+    right: 24px;
+  }
+`;
+
+
+
+const Panel = styled.div`
+  background-color: white;
+  display: block;
+  max-height: ${props => (props.whiteColor ? 0 : '500px')};
+  overflow: hidden;
+  padding: 0;
+  transition: max-height 0.25s linear;
+`;
+
+const PanelContainer = styled.div`
+  padding: 0 16px 16px 84px;
+  font-size: 14px;
+`;
+
+const PostImage = styled.span`
+  display: inline-block;
+  width: 50px;
+  height: 50px;
+  vertical-align: middle;
+  line-height: 1;
+  background-position: center center;
+  background-color: #e7e7e8;
+  background-repeat: no-repeat;
+  background-size: cover;
+  border-radius: 50%;
+  margin-right: 16px;
+  background-image: url(${props => (props.backgroundImage ? props.backgroundImage : null)});
+`;
+
+const PostTitle = styled.span`
+  font-family: 'Raleway', sans-serif;
+  font-weight: 600;
+`;
+
+const PostLearnMore = styled.a`
+  color: #00878b;
```

Comment:
I added styled components. I'm liking it, but I need to clean up how i'm including them in the component.